tbump/main.py

```diff
@@ -19,7 +19,7 @@
 from tbump.hooks import HooksRunner
 
 
-TBUMP_VERSION = "6.1.1"
```

Comment:
If changing the versioning, it might deserve its own commit... or is it an accidental change?

Reply:
It is accidental - there was a buggy test involved. Good catch